js/indicators/aroon.src.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+import H from '../parts/Globals.js';
+import multipleLinesMixin from '../mixins/multipe-lines.js';
+
+// Utils
+
+// Index of element with minimal value from array
+function minIndexInArray(arr) {
+    var minValue = Math.min.apply(null, arr);
+    return arr.lastIndexOf(minValue);
+}
+
+// Index of element with maximal value from array
+function maxIndexInArray(arr) {
+    var maxValue = Math.max.apply(null, arr);
+    return arr.lastIndexOf(maxValue);
+}
+
+
+H.seriesType('aroon', 'sma',
+    /**
+     * Aroon. This series requires the `linkedTo` option to be
+     * set and should be loaded after the `stock/indicators/indicators.js`.
+     *
+     * @extends plotOptions.sma
+     * @product highstock
+     * @sample {highstock} stock/indicators/aroon
+     *                     Aroon
+     * @since 7.0.0
+     * @excluding
+     *             allAreas,colorAxis,compare,compareBase,joinBy,keys,stacking,
+     *             showInNavigator,navigatorOptions,pointInterval,
+     *             pointIntervalUnit,pointPlacement,pointRange,pointStart
+     * @optionparent plotOptions.aroon
+     */
+    {
+        /**
+         * Paramters used in calculation of aroon series points.
+         * @excluding periods, index
+         */
+        params: {
+            /**
+             * Period for Aroon indicator
+             *
+             * @default 14
```

Comment:
No need to define default, when it's set below. Similar to type.

Reply:
Similar with `since` - when defined on the parent, no need to define for all children (needs verification).